packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -138,12 +140,13 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       const newItems = focusedItemIndices.includes(draggedIndex)
         ? (getSelectedItems() as T[])
         : [selectedItems[draggedIndex]];
-      _dropItemsAt(insertIndex, newItems);
+      _dropItemsAt(newItems);
     }
   };
 
   const _onDragStart = (item?: any, itemIndex?: number, tempSelectedItems?: any[], event?: DragEvent): void => {
-    const draggedItemIndex = itemIndex ? itemIndex! : -1;
+    /* eslint-disable-next-line eqeqeq */
+    const draggedItemIndex = itemIndex != null ? itemIndex! : -1;
```

Comment:
what does the ! for itemIndex**!** do?

Reply:
it says yes really use this I believe. I needed it because it's got the ? on it. 

P.S. I'm great at technical descriptions

Reply:
Interesting! Don't think I've used this before thanks for the explanation :P